src/components/Filter/index.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import Filter from ".";
+
+describe("Filter Component", () => {
+    it("Calling callback one time exactly in each click", () => {
```

Comment:
There is a reason this function is named `it`: it is supposed to be the first word in an English sentence starting with "it", each test description should be an English sentence starting with "it"
```suggestion
    it('calls callback "onFilter" whenever it is activated by clicking on the element with "button" role', () => {
```
Please fix other tests descriptions too

Reply:
Awesome! Thank you